### PR TITLE
make from_history_id an integer

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
:eyes: @jeremylenz

File: plugins/modules/content_export_library.py [hunk 1]
```diff
@@ -103,7 +103,7 @@
     organization: "Default Organization"
     destination_server: "airgapped.example.com"
     incremental: true
-    from_history_id: "12345"
+    from_history_id: 12345
```
Comment (evgeni) [inline plugins/modules/content_export_library.py:106]: funnily, this was the only example using a string, the others were using ints ;)

**Review by mdellweg — APPROVED** [review]
Ansible-lint does not look at the examples section? tztztz

**Comment by evgeni** [comment]
> Ansible-lint does not look at the examples section? tztztz

Neither does `ansible-test sanity`, it's essentially the wild west down there.

**Pull request merged**